packages/sdk/src/Tools/SingleAssetDepositQueue.ts

```diff
@@ -0,0 +1,297 @@
+import * as Abis from "@enzymefinance/abis";
+import { type Address, type Client, encodeFunctionData } from "viem";
+import { readContract } from "viem/actions";
+import { Viem } from "../Utils.js";
+
+//--------------------------------------------------------------------------------------------
+// DEPLOYMENT HELPERS
+//--------------------------------------------------------------------------------------------
+
```

Comment:
I would add the `deployProxy` function here, so that it's easier to find. Under the hood, it can use the DispatcherOwnerBeaconProxy code.